posts/2023-10-31-23.0.0.11-beta.adoc

```diff
@@ -0,0 +1,179 @@
+---
+layout: post
+title: "New vendor metrics available in Open Liberty 23.0.0.11-beta"
+# Do NOT change the categories section
+categories: blog
+author_picture: https://avatars3.githubusercontent.com/lauracowen
+author_github: https://github.com/lauracowen
+seo-title: New vendor metrics available in Open Liberty 23.0.0.11-beta - OpenLiberty.io
+seo-description: With MicroProfile Metrics 5.0, you can now add to your dashboard metrics, such as "REST Elapsed Time per Request", without needing to calculate them yourself.
+blog_description: With MicroProfile Metrics 5.0, you can now add to your dashboard metrics, such as "REST Elapsed Time per Request", without needing to calculate them yourself.
+open-graph-image: https://openliberty.io/img/twitter_card.jpg
+open-graph-image-alt: Open Liberty Logo
+---
+= New vendor metrics available in Open Liberty 23.0.0.11-beta
+Laura Cowen <https://github.com/lauracowen>
+:imagesdir: /
+//Blank line here is necessary before starting the body of the post.
+
+The link:/[Open Liberty] 23.0.0.11-beta contains some <<mpmetrics, new vendor metrics for MicroProfile Metrics 5.0>> (along with link:/docs/latest/reference/feature/feature-overview.html[all GA features]). You can now add to your dashboard metrics, such as "REST Elapsed Time per Request", without needing to calculate them yourself.
+
+In addition, Jakarta Data is still in beta; find out more in the link:/blog/2023/09/26/23.0.0.10-beta.html#data[Open Liberty 23.0.0.10-beta blog post].
+
+See also link:{url-prefix}/blog/?search=beta&key=tag[previous Open Liberty beta blog posts].
+
+// // // // DO NOT MODIFY THIS COMMENT BLOCK <GHA-BLOG-TOPIC> // // // // 
+// Blog issue: https://github.com/OpenLiberty/open-liberty/issues/26406
+// Contact/Reviewer: pgunapal
+// // // // // // // // 
+[#mpmetrics]
+== New vendor metrics for MicroProfile Metrics 5.0
+
+This update to MicroProfile Metrics 5.0 (`mpMetrics-5.0`) on Open Liberty includes some new link:/docs/latest/metrics-list.html#_base_and_vendor_metrics[vendor metrics] at the `/metrics` endpoint.  
+
+Previously, you could calculate the new metrics for yourself from the `Time` and `Total` counts that were already provided for various monitoring components. For example, to obtain a "response time per request" metric, you would calculate it using the array of time series data provided by the MicroProfile Metrics feature. However, not all monitoring tools support such complex time series expressions.
+
+You can use the new metrics directly in dashboards, without additional computation, with various different monitoring tools, when using the MicroProfile Metrics 5.0 feature.
```

Comment:
```suggestion
With the MicroProfile Metrics 5.0 feature, you can use the new metrics directly in the dashboards for various different monitoring tools, without any additional computation.
```

Reply:
should it be 'the dashboards **of** various monitoring tools' instead?